worker/autowithdraw.js

```diff
@@ -42,7 +42,15 @@ export async function autoWithdraw ({ data: { id }, models, lnd }) {
 
   if (pendingOrFailed.exists) return
 
-  const { invoice, wallet } = await createInvoice(id, { msats, description: 'SN: autowithdrawal', expiry: 360 }, { models })
+  const { invoice, wallet } = await createInvoice(id,
+    {
+      msats,
+      description: 'SN: autowithdrawal',
+      expiry: 360,
+      fallbackToSN: false, // if the user doesn't have an attached wallet, there is nothing to withdraw to
+      direct: true // no need to wrap this one
```

Comment:
this is to disable wrapping, that would otherwise happen automatically because the destination is an user wallet